### PR TITLE
Launcher converter cleanup

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:
Rename the suite test (from api to converter).
Cleanup unused functions and constants.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by ezrasilvera** [comment]
@EdDev Should the `RunSpecs(t, "Api Suite")`  be renamed as well ?  In converter_suite_test.go   ?

**Comment by EdDev** [comment]
> @EdDev Should the `RunSpecs(t, "Api Suite")` be renamed as well ? In converter_suite_test.go ?

Done.

File: pkg/virt-launcher/virtwrap/converter/converter_suite_test.go [hunk 1]
```diff
@@ -12,5 +12,5 @@ import (
 func TestApi(t *testing.T) {
```
Comment (AlonaKaplan) [inline pkg/virt-launcher/virtwrap/converter/converter_suite_test.go:12]: Please change the func name as well.

Comment (EdDev) [reply pkg/virt-launcher/virtwrap/converter/converter_suite_test.go]: Done.

**Pull request merged**